src/include/sof/lib/uuid.h

```diff
@@ -8,7 +8,7 @@
 #ifndef __SOF_LIB_UUID_H__
 #define __SOF_LIB_UUID_H__
 
-#include <stdint.h>
```

Comment:
It should stay here, cos this file uses `uint32_t` `uintptr_t`.
File should include what it is actually used without assumptions that it's going to be included somewhere else.

Reply:
Unless "somewhere else" is the "common" header that guarantees to deliver stddef by design of the header structure. Then you just include "common" in the units and stop including any std* things.

Reply:
After @tlauda cleaned up headers, most of them follow the convention that I wrote. To be consistent we can now include this file here or remove std* includes in all other files. I guess it's better to just include it here.